src/diff.ts

```diff
@@ -36,7 +36,8 @@ export type Diff = {
 
 export type BundleBudget = { name: string; budget: number };
 
-const ASSET_NAME_REGEXP = /^(?<assetname>[a-zA-Z0-9\.\-_]+)\.([a-zA-Z0-9]{20})\.(?<extension>js|css)$/;
+const ASSET_NAME_REGEXP =
```

Comment:
having a hard time telling the difference here: this and below were just formatting changes, right?

Reply:
Oh right sorry, should have commented. Yes, they're unchanged. This is just a result of upgrading prettier.